modules/cborm/models/criterion/Restrictions.cfc

```diff
@@ -169,8 +169,21 @@ component singleton{
 		return restrictions.sizeNE(arguments.property, arguments.propertyValue);
 	}
 	
-	// Use arbitrary SQL to modify the resultset
-	any function sqlRestriction(required string sql){
+	any function sqlRestriction(required string sql, any parameters ){
+        //if we defined a parameter then its assume its a string and pass it through right
```

Comment:
Could there be any other types in stead of just string? What about numeric, floats, etc?

Reply:
I have merged this manually into be, but have added full support for all types.